sacro/views.py

```diff
@@ -110,7 +110,9 @@ def contents(request):
 
 @require_http_methods(["POST"])
 def review(request):
-    outputs = get_outputs(request.POST)
+    breakpoint
+    # we load the path from the querystring, even though this is a post request
+    outputs = get_outputs(request.GET)
```

Comment:
Why are we passing in the path via a querystring?

Reply:
Because otherwise it would need to be in the path, and it's an absolute windows file path, to that's awkward.  The path would look like:

`/review/c/users/somedir/otherdir/outputs/results.json`

And need platform specific decoding. Passing path on via the query string bypasses all that.

Could also do sthlike `/review/BASE64ENCODEDPATH`, but that's difficult for a dev to manually create when testing.

We could also put it in a cookie, but that has same dev unfriendlyness issue, and means that it becomes stateful, which has other implications and is generally a bad idea.